src/sentry/static/sentry/app/stores/groupingStore.tsx

```diff
@@ -9,19 +10,146 @@ import {
 } from 'app/actionCreators/indicator';
 import GroupingActions from 'app/actions/groupingActions';
 
-const api = new Client();
-
 // Between 0-100
 const MIN_SCORE = 0.6;
 
 // @param score: {[key: string]: number}
-const checkBelowThreshold = scores => {
-  const scoreKeys = (scores && Object.keys(scores)) || [];
+const checkBelowThreshold = (scores = {}) => {
+  const scoreKeys = Object.keys(scores);
   return !scoreKeys.map(key => scores[key]).find(score => score >= MIN_SCORE);
 };
 
-const GroupingStore = Reflux.createStore({
+type State = {
+  // List of fingerprints that belong to issue
+  mergedItems: [];
+  // Map of {[fingerprint]: Array<fingerprint, event id>} that is selected to be unmerged
+  unmergeList: Map<any, any>;
```

Comment:
Would `Record<string, any>` work here?

Reply:
maybe ... but I prefer to leave like this for now ... I might update it in this follow up PR https://github.com/getsentry/sentry/pull/21607